.gitignore

```diff
@@ -21,3 +21,6 @@ sftp-config.json
 # Only apps should have lockfiles
 yarn.lock
 package-lock.json
+
+# Visual Code folder
+.vscode
```

Comment:
please revert this; specific IDE files should go in your personal global gitignore, and not clutter every repo you use.